infrastructure/validators/operational-tasks/how-to-upgrade-validator.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Upgrade Your Node
+description: Instructions on how to upgrade your validator node.
+---
+
+Validators perform critical functions for the network by [backing and including blocks](https://wiki.polkadot.network/docs/learn-parachains-protocol){target=_blank}. Validators may have to go offline for short-periods of time to upgrade client software or to upgrade the host machine.
+
+Usually, standard client upgrades will only require you to stop the service, replace the binary and
+restart the service. This operation can be executed within a session.
+
+Validators may also need to perform long-lead maintenance tasks that will span more than one
+session. Under these circumstances, an active validator may chill their stash and be removed from
+the active validator set. Alternatively, the validator may substitute the active validator server
+with another allowing the former to undergo maintenance activities.
+
+This guide will provide an option for validators to seamlessly substitute an active validator server
+to allow for maintenance operations.
+
+The process can take several hours, so make sure you understand the instructions first and plan
+accordingly.
+
+!!!tip "Keep an eye out on new releases from the community"
+    Upgrade or downgrade accordingly.
+
+
+## Key Components
+
+### Session Keys
+
+Session keys are stored in the client and used to sign validator operations. They are what link your
+validator node to your staking proxy. If you change them within a session you will have to wait for
+the current session to finish and a further two sessions to elapse before they are applied.
+
+[More info about keys in Polkadot.](https://wiki.polkadot.network/docs/learn-cryptography#keys){target=_blank}
+
+### Keystore
+
+Each validator server contains essential private keys in a folder called the _keystore_. These keys
+are used by a validator to sign transactions at the network level. If two or more validators sign
+certain transactions using the same keys, it can lead to an [equivocation slash.](https://wiki.polkadot.network/docs/learn-offenses){target=_blank}
```

Comment:
```suggestion
certain transactions using the same keys, it can lead to an [equivocation slash.](https://wiki.polkadot.network/docs/learn-offenses){target=\_blank}
```